src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialogView.java

```diff
@@ -48,7 +48,8 @@
     private final BibDatabaseMode mode;
     private final BibEntryTypesManager entryTypesManager;
 
-    @FXML private TableView<EntryTypeViewModel> entryTypes;
+    @FXML
+    private TableView<EntryTypeViewModel> entryTypes;
```

Comment:
Please try to avoid introducing whitespaces or linebreaks without need.
Keep a consistent code style in the code base.